apps/judicial-system/web/src/components/InfoCard/InfoCardCaseScheduledIndictment.tsx

```diff
@@ -1,33 +1,67 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
 
-import { Institution } from '../../graphql/schema'
+import { Text } from '@island.is/island-ui/core'
+
+import { IndictmentDecision, Institution } from '../../graphql/schema'
 import InfoCard from './InfoCard'
 import InfoCardCaseScheduled from './InfoCardCaseScheduled'
 import { strings } from './InfoCardCaseScheduledIndictment.strings'
 
 interface Props {
   court: Institution
+  indictmentDecision?: IndictmentDecision | null
   courtDate: string
   courtRoom?: string | null
   postponedIndefinitelyExplanation?: string | null
 }
 
 const InfoCardCaseScheduledIndictment: React.FC<Props> = (props) => {
-  const { court, courtDate, courtRoom, postponedIndefinitelyExplanation } =
-    props
+  const {
+    court,
+    indictmentDecision,
+    courtDate,
+    courtRoom,
+    postponedIndefinitelyExplanation,
+  } = props
   const { formatMessage } = useIntl()
 
-  return postponedIndefinitelyExplanation ? (
-    <InfoCard
-      data={[
-        {
-          title: formatMessage(strings.postponed),
-          value: postponedIndefinitelyExplanation,
-        },
-      ]}
-      icon="calendar"
-    />
+  return indictmentDecision ? (
+    indictmentDecision === IndictmentDecision.POSTPONING ? (
+      <InfoCard
+        data={[
+          {
+            title: formatMessage(strings.postponingTitle),
+            value: (
+              <Text marginTop={2}>{postponedIndefinitelyExplanation}</Text>
+            ),
+          },
+        ]}
+        icon="calendar"
+      />
+    ) : indictmentDecision === IndictmentDecision.POSTPONING_UNTIL_VERDICT ? (
+      <InfoCard
+        data={[
+          {
+            title: formatMessage(strings.schedulingUntilVerdictTitle),
+            value: (
+              <Text marginTop={2}>
+                {formatMessage(strings.schedulingUntilVerdictText)}
+              </Text>
+            ),
+          },
+        ]}
+        icon="calendar"
+      />
+    ) : (
+      // indictmentDecision === IndictmentDecision.SCHEDULING
```

Comment:
Is this supposed to be here?

Reply:
Yes, it tells us what value we are expecting. I want this component rewritten at a later date to make it clearer.